kafka-versions.yaml

```diff
@@ -8,15 +8,14 @@
 
 # Format of this file:
 #   <version> is the kafka version number
-#   <default> when `true` this is the version to be used by the CO by default when a `Kafka` resource lacks an explicit version. 
-#             Only one Kafka version can be marked as default.
+#   <default> when `true` this is the version to be used by the CO by default when a `Kafka` resource lacks an explicit version. Only one Kafka version can be marked as default.
 #   <protocol> is the default `inter.broker.protocol.version` used by this Kafka version
 #   <format> is the default `log.message.format.version` used by this Kafka version
-#   <checksum> is the SHA512 checksum of the Kafka binary downloaded from apache.org
+#   <url> is the remote address of the hosted Kafka binary tar archive for this version of Kafka. Either this or 'file' should be set.
```

Comment:
Can't we use just one field for both file and URL? It is quite common to represent a local file as a URL with `file://...`.

Reply:
We could do this, it would require some string munging in the `kafka-version-tools.sh` script to parse the URL. I think originally we didn't do this as we wanted to just use a common `curl` command (so the file would have to be local hosted). But now we have the file copy logic in there anyway it may not be an issue. @tombentley what do you think?

Reply:
Actually, `curl` seems to understand `file://`. At least on my MacOS I can do `curl file:///Users/kuba/development/kubernetes-client/circle.yml` (Where in my case the file `/Users/kuba/development/kubernetes-client/circle.yml` actually exists). So I wonder if one URL makes one less path in the code to implement / test. You will download it from some location into the temp, checksum and unpact it etc.

Reply:
Well would you look at that it does! Wish I had checked that earlier. Ok I have changed it to be a single `url` field in the versions file with either `http://` or `file://` this removes several if statements and is easier to explain in the docs.